docs/OpenCLIExamples.rst

```diff
@@ -0,0 +1,346 @@
+==========================
+CLI Open Endpoint Examples
+==========================
+
+The HCA CLI provides users of the Human Cell Atlas (HCA) to access and download data sets from the HCA. This page
+covers how to access the HCA using the HCA command line utility.
+
+*NOTE:* The HCA CLI utility is compatible with Python 3.5+.
+
+
+``hca create-version``
+----------------------
+
+Returns a timestamp in ``DSS_VERSION`` format (e.g., ``1985-04-12T232050.520000Z``), useful for
```

Comment:
`useful` -> `necessary`

Reply:
Fixed in both OpenAPI and OpenCLI documents